products-service/handler-unique.js

```diff
@@ -1,30 +1,45 @@
 'use strict';
-import productList from './statics/productList.json';
+import { client } from './db-connection';
 
 const headers = {
   'Content-Type': 'application/json',
   'Access-Control-Allow-Origin': '*',
 };
 
 const getProductById = async (event) => {
-  const id = event.queryStringParameters.id;
-  const selectedProduct = productList.find(function (product) {
-    return id === product.id;
-  });
-  const [statusCode, product] =
-    typeof selectedProduct === 'undefined'
-      ? [404, 'Product not Found']
-      : [200, selectedProduct];
+  try {
+    console.log(
+      'Getting a unique product by id - on  GET /products/{id} endpoint'
+    );
+    const id = event.queryStringParameters.id;
```

Comment:
First I couldn't understand why I get error when trying to reach a new product with path like 
[https://hih14h2qbi.execute-api.us-east-1.amazonaws.com/products/0b41f7ce-c012-4ddf-970a-eeb99ed4f6fa](https://hih14h2qbi.execute-api.us-east-1.amazonaws.com/products/0b41f7ce-c012-4ddf-970a-eeb99ed4f6fa)

But now I see. Any reason why you decided to use query parameters instead of `event.pathParameters`?